Description: set rpath on chromium-browser and v8_context_snapshot_generator executables so that they can locate libffmpeg.so
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Bug-Ubuntu: https://launchpad.net/bugs/1718885

--- a/src/chrome/BUILD.gn
+++ b/src/chrome/BUILD.gn
@@ -387,6 +387,8 @@ if (!is_android && !is_mac) {
           "app/chrome_main_linux.h",
         ]
       }
+
+      configs += [ "//build/config/gcc:rpath_for_built_shared_libraries" ]
     }
   }
   _chrome_exe("chrome_initial") {
--- a/src/tools/v8_context_snapshot/BUILD.gn
+++ b/src/tools/v8_context_snapshot/BUILD.gn
@@ -89,6 +89,10 @@ if (use_v8_context_snapshot) {
         "//v8",
       ]
 
+      if (is_linux) {
+        configs += [ "//build/config/gcc:rpath_for_built_shared_libraries" ]
+      }
+
       configs += [
         "//v8:external_startup_data",
         ":disable_icf",
--- a/src/components/nacl/loader/BUILD.gn
+++ b/src/components/nacl/loader/BUILD.gn
@@ -103,6 +103,7 @@ if ((is_linux || is_chromeos) && current
       "nacl_helper_linux.h",
     ]
 
+    configs += [ "//build/config/gcc:rpath_for_built_shared_libraries" ]
     deps = [
       ":loader",
       "//base",
